clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1032,25 +1036,89 @@ raw2trace_t::process_syscall_pt(raw2trace_thread_data_t *tdata, uint64_t syscall
     }
     trace_entry_t end_entry = { .type = TRACE_TYPE_MARKER,
                                 .size = TRACE_MARKER_TYPE_SYSCALL_TRACE_END,
-                                .addr = 0 };
+                                .addr = sysnum };
```

Comment:
The docs for this marker need to be updated to say what the value holds.

Reply:
Done.